homeassistant/components/tankerkoenig/config_flow.py

```diff
@@ -144,6 +144,30 @@ async def async_step_select_station(
             options={CONF_SHOW_ON_MAP: True},
         )
 
+    async def async_step_reauth(self, data: dict[str, Any]) -> FlowResult:
```

Comment:
`async_step_reauth` should be using `Mapping[str, Any]` for the argument (see #72385).

```suggestion
    async def async_step_reauth(self, data: Mapping[str, Any]) -> FlowResult:
```